roles/neutron-common/defaults/main.yml

```diff
@@ -1,6 +1,18 @@
 ---
 neutron:
   l2_population: False
+  vxlan:
+    # The multicast prefix to use for broadcast, multicast and unknown frames
+    # Consider using a larger prefix (e.g. a /24) on deployments with a
+    # dedicated network infrastructure between nodes. This is a compromise
+    # between network resource allocation, and efficient delivery to BUM
+    # frames.
+    group_prefix: 239.1.1.1
+    # If using multicast, limit the L3 diameter of VXLAN multicast traffic by
+    # specifying the TTL on VXLAN packets. By default with limit the TTL to
```

Comment:
Also, is there a scenario in which we would want to increase the ttl? How would an operator/deployer make that decision?